plugins/obs-ffmpeg/obs-ffmpeg-mpegts.c

```diff
@@ -437,11 +437,11 @@ static inline int allocate_custom_aviocontext(struct ffmpeg_output *stream,
 	if (is_rist)
 		s = avio_alloc_context(
 			buffer, buffer_size, AVIO_FLAG_WRITE, h, NULL,
-			(int (*)(void *, uint8_t *, int))librist_write, NULL);
+			(int (*)(void *, const uint8_t *, int))libsrt_write, NULL);
```

Comment:
You changed `librist_write` to `libsrt_write` in the conditional block for `is_rist`. Revert that change.